.vib/chartmuseum/goss/chartmuseum.yaml

```diff
@@ -0,0 +1,16 @@
+group:
+  harbor:
+    exists: true
+user:
+  harbor:
+    exists: true
+file:
+  /bitnami/data:
+    exists: true
+    mode: "0775"
+    filetype: directory
```

Comment:
If there is now need to check the owner/group of the path we should use the goss templates instead of making this file bigger.